### PR TITLE
Added F# iOS Sample

### PR DESCRIPTION
**Pull request by delneg**
Continuation of https://github.com/dotnet/net6-mobile-samples/issues/88

Decided to try it out and it worked out pretty easily.

Screenshot 2021-03-14 at 19 34 59

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/net6-mobile-samples?pullRequest=97) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/net6-mobile-samples?pullRequest=97) before we can accept your contribution.<br/><br/>:x: delneg [sign now](https://cla.dotnetfoundation.org/dotnet/net6-mobile-samples?pullRequest=97)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/net6-mobile-samples?pullRequest=97) it.</sub>

File: HelloiOSFSharp/HelloiOSFSharp.fsproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <PropertyGroup>
+    <TargetFramework>net6.0-ios</TargetFramework>
+    <RuntimeIdentifier>ios-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json" />
+    <InterfaceDefinition Include="LaunchScreen.storyboard" />
+    <None Include="Info.plist" />
+    <None Include="Entitlements.plist" />
```
Comment (jonathanpeppers) [inline HelloiOSFSharp/HelloiOSFSharp.fsproj:11]: If you remove these, does everything still work?

A lot of files are wildcarded by default in .NET 6, I think it's seeing these twice:
```
LaunchScreen.storyboard : error : The file 'LaunchScreen.storyboard' conflicts with 'LaunchScreen.storyboard'.
```

Comment (delneg) [reply HelloiOSFSharp/HelloiOSFSharp.fsproj]: Yes, it does work.
I've pushed the new version

**Review by jonathanpeppers — APPROVED** [review]
Thanks! I'll merge if my latest two commits didn't break anything.

I think we should put this in a `fsharp` folder, so that we can eventually have a `csharp` folder, too.

**Comment by delneg** [comment]
I can confirm it didn't break anything,
` dotnet build fsharp/HelloiOSFSharp -t:Run` ran fine.

Thanks!

**Pull request merged**